include/mbedtls/aes.h

```diff
@@ -255,30 +264,30 @@ int mbedtls_aes_crypt_ctr( mbedtls_aes_context *ctx,
 #endif /* MBEDTLS_CIPHER_MODE_CTR */
 
 /**
- * \brief           Internal AES block encryption function
- *                  (Only exposed to allow overriding it,
- *                  see MBEDTLS_AES_ENCRYPT_ALT)
+ * \brief Internal AES block encryption function. This is 
+ *        only exposed to allow overriding it,
+ *        see MBEDTLS_AES_ENCRYPT_ALT.
  *
- * \param ctx       AES context
- * \param input     Plaintext block
- * \param output    Output (ciphertext) block
+ * \param ctx       The AES context to encrypt.
+ * \param input     The plaintext block.
+ * \param output    The output (ciphertext) block.
  *
- * \return          0 if successful
+ * \return          Zero if successful.
  */
 int mbedtls_internal_aes_encrypt( mbedtls_aes_context *ctx,
                                   const unsigned char input[16],
                                   unsigned char output[16] );
 
 /**
- * \brief           Internal AES block decryption function
- *                  (Only exposed to allow overriding it,
- *                  see MBEDTLS_AES_DECRYPT_ALT)
+ * \brief Internal AES block decryption function. This is
+ *        only exposed to allow overriding it,
+ *        see MBEDTLS_AES_DECRYPT_ALT.
```

Comment:
if we are changing, I would change to:
"This is only exposed to allow overriding it, using MBEDTLS_AES_DECRYPT_ALT."

Reply:
done